### PR TITLE
[type:bugfix] fix grpc ci error

### PR DESCRIPTION
**Pull request by Aias00**
fix grpc ci error
image

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by Aias00** [comment]
for fixing ci error like this:

image

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/register/AbstractShenyuClientRegisterServiceImpl.java [hunk 1]
```diff
@@ -64,7 +66,9 @@
  * Abstract strategy.
  */
 public abstract class AbstractShenyuClientRegisterServiceImpl extends FallbackShenyuClientRegisterService implements ShenyuClientRegisterService {
-
+    
+    private final Logger logger = LoggerFactory.getLogger(AbstractShenyuClientRegisterServiceImpl.class);
```
Comment (moremind) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/service/register/AbstractShenyuClientRegisterServiceImpl.java:70]: use LOG

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterGrpcServiceImpl.java [hunk 2]
```diff
@@ -75,6 +79,7 @@ protected String ruleHandler() {
     @Override
     protected void registerMetadata(final MetaDataRegisterDTO metaDataDTO) {
         MetaDataService metaDataService = getMetaDataService();
+        LOG.info("grpc register metadata:{}", GsonUtils.getInstance().toJson(metaDataDTO));
```
Comment (moremind) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterGrpcServiceImpl.java:82]: may debug level?

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterGrpcServiceImpl.java [hunk 3]
```diff
@@ -140,6 +145,7 @@ public String offline(final String selectorName, final List<URIRegisterDTO> offl
                 removeDiscoveryUpstream(selectorDO.getId(), grpcUpstream.getUpstreamUrl());
             }
             DiscoverySyncData discoverySyncData = fetch(selectorDO.getId(), selectorDO.getName(), pluginName);
+            LOG.info("grpc offline discoverySyncData:{}", GsonUtils.getInstance().toJson(discoverySyncData));
```
Comment (moremind) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/service/register/ShenyuClientRegisterGrpcServiceImpl.java:148]: may debug level?

**Review by moremind — CHANGES_REQUESTED** [review]

**Review by moremind — APPROVED** [review]

**Review by moremind — APPROVED** [review]

**Pull request merged**